infra/k8s/tools/README.md

```diff
@@ -0,0 +1,9 @@
+# Tools and commands that are helpful during running Testground on Kubernetes
+
+1. `confirm_weave.sh`
```

Comment:
@raulk we might want to move this to a command, such as `testground validate-cluster` or something like this, which does a set of checks that the cluster is setup correctly:
- confirms that `sidecar` is running on every host.
- confirms that `redis` is running
- confirms that routes on hosts are setup correctly

`kops` has a similar tool:

```
➜  testground git:(less-steps) kops validate cluster simple.k8s.local
Validating cluster simple.k8s.local

INSTANCE GROUPS
NAME                    ROLE    MACHINETYPE     MIN     MAX     SUBNETS
master-eu-central-1a    Master  c5.2xlarge      1       1       eu-central-1a
nodes                   Node    c5.2xlarge      14      14      eu-central-1a

NODE STATUS
NAME                                            ROLE    READY
ip-172-20-34-37.eu-central-1.compute.internal   node    True
ip-172-20-39-154.eu-central-1.compute.internal  master  True
ip-172-20-39-244.eu-central-1.compute.internal  node    True
ip-172-20-45-108.eu-central-1.compute.internal  node    True
ip-172-20-48-175.eu-central-1.compute.internal  node    True
ip-172-20-52-192.eu-central-1.compute.internal  node    True
ip-172-20-53-4.eu-central-1.compute.internal    node    True
ip-172-20-58-197.eu-central-1.compute.internal  node    True
ip-172-20-58-89.eu-central-1.compute.internal   node    True
ip-172-20-59-73.eu-central-1.compute.internal   node    True
ip-172-20-60-196.eu-central-1.compute.internal  node    True
ip-172-20-61-105.eu-central-1.compute.internal  node    True
ip-172-20-61-144.eu-central-1.compute.internal  node    True
ip-172-20-61-26.eu-central-1.compute.internal   node    True
ip-172-20-63-3.eu-central-1.compute.internal    node    True

Your cluster simple.k8s.local is ready
```